### ISSUE TITLE
Loss when training with my own data

### ISSUE THREAD

**Issue by Sissi-lu** [YvanYin/Metric3D#177]
Hi, thank you for sharing the code and data you used in Metric3D. It helps me a lot to understand your work. But when I trained with my own data with RGB-Depth-Normal label following by [#105 ](https://github.com/YvanYin/Metric3D/issues/105)， I felt confused about what kind of loss I should apply.

In the picture, I saw you mainly use three losses, .i.e $L_d$ for supervising the depth, $L_n$ for supervising normal, and $L_{d-n}$ to align the predicted depth：
<img width="474" alt="image" src="https://github.com/user-attachments/assets/cfa6dd33-1188-419b-9c6a-bdef4bec9929">
Specifically, the $L_{d}$ is composed by:
<img width="356" alt="image" src="https://github.com/user-attachments/assets/4cc0ce26-3353-4c2d-a443-d6f29b7d55dc">
but it seems that $L_{PWN}$ and $L_{VNL}$ are the normal loss?

When I tried to understand this loss in your code, I found they are hard to correspond to. This brings me these main questions:
1、Can you tell me the specific meaning of the loss in your paper, as well as their corresponding sense? And what do they correspond to the code?
2、When I want to fine-tune the model with my own data with GT of depth and normal, which loss should I apply? I'm just too confused about the loss in code. Now I use the configuration loss of `vit.raft5.giant2.nyu.py`,  .i.e,
`losses=dict(
    decoder_losses=[
        dict(type='VNLoss', sample_ratio=0.2, loss_weight=1.0),   
        dict(type='GRUSequenceLoss', loss_weight=1.0, loss_gamma=0.9, stereo_sup=0),
        dict(type='NormalBranchLoss', loss_weight=1.5, loss_fn='NLL_ours_GRU'),
        dict(type='DeNoConsistencyLoss', loss_weight=0.001, loss_fn='CEL', scale=2),
        dict(type='HDNRandomLoss', loss_weight=0.5, random_num=10),
        dict(type='HDSNRandomLoss', loss_weight=0.5, random_num=20, batch_limit=4),
        dict(type='PWNPlanesLoss', loss_weight=1),
    ],
)` 
I find some of them equal to zero or never reduce, just as follows:
![image](https://github.com/user-attachments/assets/61141e02-26fc-4ecd-aa71-b1436551e482)

Thanks a lot!
